tools/python/maps_generator/README.md

```diff
@@ -8,292 +8,96 @@ maps built by a generator_tool newer than the app.**
 
 ```sh
 cd tools/python
-python -m maps_generator --countries="Canada_Yukon_North, Canada_Yukon_Whitehorse" --skip="Coastline"
 ```
 
-## Help
-
+Build two mwm files without using coastlines:
 ```sh
-python$ python -m maps_generator -h
+python -m maps_generator --countries="Canada_Yukon_North, Canada_Yukon_Whitehorse" --skip="Coastline"
 ```
 
-```
-usage: __main__.py [-h] [--config CONFIG] [-c [CONTINUE]]
-                   [--countries COUNTRIES] [--skip SKIP]
-                   [--from_stage FROM_STAGE] [--production]
-
-Tool for generation maps for maps.me application.
-
-optional arguments:
-  -h, --help            show this help message and exit
-  --config CONFIG       Path to config
-  -c [CONTINUE], --continue [CONTINUE]
-                        Continue the last build or specified in CONTINUE from
-                        the last stopped stage.
-  -s SUFFIX, --suffix SUFFIX
-                        Suffix of the name of a build directory.
-  --countries COUNTRIES
-                        List of regions, separated by a comma or a semicolon,
-                        or path to file with regions, separated by a line
-                        break, for which maps will be built. The names of the
-                        regions can be seen in omim/data/borders. It is
-                        necessary to set names without any extension.
-  --without_countries WITHOUT_COUNTRIES
-                        List of regions to exclude them from generation.
-                        Syntax is the same as for --countries.
-  --skip SKIP           List of stages, separated by a comma or a semicolon,
-                        for which building will be skipped. Available skip
-                        stages: DownloadAndConvertPlanet, UpdatePlanet,
-                        Coastline, Preprocess, Features, Mwm, Index,
-                        CitiesIdsWorld, Ugc, Popularity, Srtm, IsolinesInfo,
-                        Descriptions, Routing, RoutingTransit, CountriesTxt,
-                        ExternalResources, LocalAds, Statistics, Cleanup.
-  --from_stage FROM_STAGE
-                        Stage from which maps will be rebuild. Available
-                        stages: DownloadAndConvertPlanet, UpdatePlanet,
-                        Coastline, Preprocess, Features, Mwm, Index,
-                        CitiesIdsWorld, Ugc, Popularity, Srtm, IsolinesInfo,
-                        Descriptions, Routing, RoutingTransit, CountriesTxt,
-                        ExternalResources, LocalAds, Statistics, Cleanup.
-  --coasts              Build only WorldCoasts.raw and WorldCoasts.rawgeom
-                        files
-  --force_download_files
-                        If build is continued, files will always be downloaded
-                        again.
-  --production          Build production maps. In another case, 'osm only
-                        maps' are built - maps without additional data.
-  --order ORDER         Mwm generation order.
```

Comment:
no need to copy the `--help` output to the readme, a user can run this command himself (the readme says "To see other possible command-line options...")

Reply:
Please, keep it.

Reply:
But its not a part of the process :)
Its a list of possible advanced options, a few users will need them and there is a reference how to see them (invoke `--help`).. What's the point of having a full duplicate?

It might make sense to list or reference the most widely used ones. What are they?

Reply:
It makes sense to avoid copy-pasting of the generator output and keep details only in one place (in the code).
However, it's better to explicitly mention this (e.g. "run ... --help and check detailed instructions"), and also to mention some corner cases/params like "to do advanced things like ... please check documentation in ... --help output".

Reply:
Its mentioned already right at the end of the **Basic usage** section:
> To see other possible command-line options:
```sh
python -m maps_generator -h
```
And the `--production` option behavior is mentioned there.

What are other options that are really important to mention in the readme?